### PR TITLE
Fix main cities order in search results #219 #230

### PR DESCRIPTION
**Pull request by nicolasleger**
See https://github.com/covoiturage-libre/covoiturage-libre/issues/230

### CODE REVIEW

**Comment by webplate** [comment]
This elastic search is quite elastic indeed :) The new setting you chose allows Vanne to be selected when typing "vanne". It's a bit strange that "can" gives Canly, but "cann" gives Cannes.

Have you tried other settings ?

Maybe factor 8 is exagerated ? It was nice to have the main cities preselected with first letters ("par" was giving Paris, "mar" Marseille, and so on... It's not the case anymore with this PR.

Can we have best of both worlds ?

File: app/controllers/cities_controller.rb [hunk 1]
```diff
@@ -82,8 +82,10 @@ def autocomplete
     @results = City.search(
       params[:term],
       limit: 5,
-      boost_where: { postal_code: MAIN_POSTAL_CODES,
-        country_code: { value: 'FR', factor: 5 } },
+      boost_where: {
+        postal_code: { value: MAIN_POSTAL_CODES, factor: 8 },
```
Comment (webplate) [inline app/controllers/cities_controller.rb]: Maybe this factor can be tuned a bit better ?

Comment (nicolasleger) [reply app/controllers/cities_controller.rb]: Yes but I don't know exactly how.
Better to take inspiration from other OSS projects with City search.

**Comment by nicolasleger** [comment]
Like my comment : Better to take inspiration from other OSS projects with City search.

**Comment by nicolasleger** [comment]
However we need a rapid solution for the next update that is a least a minor plus.
Then it could be an another issue to optimize it further.

**Pull request merged**